app/serializers/invoice_item_serializer.rb

```diff
@@ -1,3 +1,7 @@
 class InvoiceItemSerializer < ActiveModel::Serializer
   attributes :id, :invoice_id, :item_id, :quantity, :unit_price
+
+  def unit_price
+    object.unit_price.to_s.chars.insert(-3, ".").join
```

Comment:
Interesting! I was imagining that we'd do this with math, but I think this will avoid rounding errors. Cool!